### PR TITLE
fix restTime bug

### PR DESCRIPTION
**Pull request by CoffeeLatte007**
### Ⅰ. Describe what this PR did
fix restTime bug

### CODE REVIEW

**Comment by CoffeeLatte007** [comment]
> I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?

why? mills is enough

**Comment by KAMIJYOUDOUMA** [comment]
> I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?

Or just change `System.currentTimeMillis()` to `System.nanoTime()`,and change the value of `GLOBAL_SESSOION_LOCK_TIME_OUT_MILLS`.

**Comment by KAMIJYOUDOUMA** [comment]
> restTime

```
long parkTime = backOffTime < restTime ? backOffTime : restTime;
LockSupport.parkNanos(parkTime);
```
The unit between `backOffTime` and `restTime` is different.

**Comment by KAMIJYOUDOUMA** [comment]
> > I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?
> 
> why? mills is enough
```
long parkTime = backOffTime < restTime ? backOffTime : restTime;
LockSupport.parkNanos(parkTime);
```
The unit between `backOffTime` and `restTime` is different.

**Comment by CoffeeLatte007** [comment]
> > restTime
> 
> ```
> long parkTime = backOffTime < restTime ? backOffTime : restTime;
> LockSupport.parkNanos(parkTime);
> ```
> 
> The unit between `backOffTime` and `restTime` is different.



> > > I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?
> > 
> > 
> > why? mills is enough
> 
> ```
> long parkTime = backOffTime < restTime ? backOffTime : restTime;
> LockSupport.parkNanos(parkTime);
> ```
> 
> The unit between `backOffTime` and `restTime` is different.

yes , i will fix it

**Comment by CoffeeLatte007** [comment]
> > > I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?
> > 
> > 
> > why? mills is enough
> 
> ```
> long parkTime = backOffTime < restTime ? backOffTime : restTime;
> LockSupport.parkNanos(parkTime);
> ```
> 
> The unit between `backOffTime` and `restTime` is different.

i fix it, please review

**Comment by KAMIJYOUDOUMA** [comment]
> > > > I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?
> > > 
> > > 
> > > why? mills is enough
> > 
> > 
> > ```
> > long parkTime = backOffTime < restTime ? backOffTime : restTime;
> > LockSupport.parkNanos(parkTime);
> > ```
> > 
> > 
> > The unit between `backOffTime` and `restTime` is different.
> 
> i fix it, please review

The `PARK_TIMES_BASE` will be 1000 times of its original value,I'm not sure whether it is right or not.

File: server/src/main/java/io/seata/server/session/GlobalSession.java [hunk 1]
```diff
@@ -511,24 +512,23 @@ public void lockAndExcute(LockRunnable excuteRunnable) throws TransactionExcepti
 
         private static final int PARK_TIMES_BASE = 10;
 
-        private static final int PARK_TIMES_BASE_NANOS = 1 * 1000 * 1000;
+        private static final int PARK_TIMES_BASE_MILLS = 1 * 1000;
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/session/GlobalSession.java]: 1 ms=10^6 ns ?

Comment (CoffeeLatte007) [reply server/src/main/java/io/seata/server/session/GlobalSession.java]: it is my mistakes, i fix it

**Comment by CoffeeLatte007** [comment]
> > > > > I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?
> > > > 
> > > > 
> > > > why? mills is enough
> > > 
> > > 
> > > ```
> > > long parkTime = backOffTime < restTime ? backOffTime : restTime;
> > > LockSupport.parkNanos(parkTime);
> > > ```
> > > 
> > > 
> > > The unit between `backOffTime` and `restTime` is different.
> > 
> > 
> > i fix it, please review
> 
> The `PARK_TIMES_BASE` will be 1000 times of its original value,I'm not sure whether it is right or not
it is my mistake, PARK_TIMES_BASE_MILLS should be  1 ms

**Comment by CoffeeLatte007** [comment]
> > > > > I think that the unit of `restTime` is wrong.Should change it to `nanosecond`?
> > > > 
> > > > 
> > > > why? mills is enough
> > > 
> > > 
> > > ```
> > > long parkTime = backOffTime < restTime ? backOffTime : restTime;
> > > LockSupport.parkNanos(parkTime);
> > > ```
> > > 
> > > 
> > > The unit between `backOffTime` and `restTime` is different.
> > 
> > 
> > i fix it, please review
> 
> The `PARK_TIMES_BASE` will be 1000 times of its original value,I'm not sure whether it is right or not.

fix it PARK_TIMES_BASE_MILLS = 1

**Pull request merged**